src/View/ViewBuilder.php

```diff
@@ -500,6 +522,34 @@ public function getOptions(): array
         return $this->_options;
     }
 
+    /**
+     * Set the config merge strategy for view options.
+     *
+     * This controls how options set via ViewBuilder are merged with
+     * the View class's default configuration.
+     *
+     * @param string $strategy The merge strategy. Can be:
```

Comment:
```suggestion
     * @param self::MERGE_DEEP|self::MERGE_SHALLOW $strategy The merge strategy. Can be:
```

phpstan supports this, we can use `@phpstan-var` instead if other IDEs don't.